packages/client/wallets/aa/src/utils/blockchain.ts

```diff
@@ -1,7 +1,18 @@
+import { getBundlerRPC } from "@/blockchain";
 import type { Deferrable } from "@ethersproject/properties";
 import { type TransactionRequest } from "@ethersproject/providers";
 import { BigNumber, BigNumberish } from "ethers";
 import { BytesLike, hexlify } from "ethers/lib/utils";
+import { createPublicClient, http } from "viem";
+
+import { EVMBlockchainIncludingTestnet } from "@crossmint/common-sdk-base";
+
+export type VerifyMessageParams = {
+    address: `0x${string}`;
+    message: string;
+    signature: `0x${string}` | Uint8Array;
+    chain: EVMBlockchainIncludingTestnet;
```

Comment:
if we do need to keep chain, this should be `Blockchain` – we've made an alias because for users of the SDK, `EVMBlockchainIncludingTestnet` is a rather unsightly type name haha

Reply:
I agree, although the issue is that we need to use a type there, and the options are:
- `EVMBlockchainIncludingTestnet`
- or create a new type ( I'd hate that 😅)

`Blockchain` or `EVMBlockchain` won't work because those don't include testnet chains.

Also, the user will see it differently:
 ![Screenshot 2024-04-26 at 11 38 51 AM](https://github.com/Crossmint/crossmint-sdk/assets/132915515/a043b4f4-eb21-4f72-aed0-eaf7be84de20)

If `EVMBlockchainIncludingTestnet` sounds bad then I vote to not export `VerifyMessageParams`. I just exported it to give to the developer a quick type if needed. 

LMK

Reply:
ah, somehow this got reverted but we need it to match the API we outlined in EDD
https://github.com/Crossmint/crossmint-sdk/pull/436/files

Reply:
I added this because in our EDD, we specifically call this type "Blockchain"
https://github.com/Crossmint/crossmint-sdk/pull/436/files
so we can import it from within the SDK itself rather than common-sdk, but we may need to move this somewhere else to make it more accessible

Reply:
Even when used as in the example, from the implementation project it's seen as the original name type

![code](https://github.com/Crossmint/crossmint-sdk/assets/132915515/e35ceb44-71ee-4f4a-b907-ac26338b0278)
![Screenshot 2024-04-26 at 1 24 56 PM](https://github.com/Crossmint/crossmint-sdk/assets/132915515/cc498522-8212-40cb-b529-fc2f2f81360d)

Reply:
hmmm ok I wonder if we need to do this in a different way then. For now we can merge this

Reply:
Sure thing, If we find a way I'll change it